src/firefly/java/edu/caltech/ipac/firefly/server/ServerContext.java

```diff
@@ -835,12 +835,9 @@ protected RequestOwner childValue(RequestOwner requestOwner) {
 
     @WebListener
     public static class ContextListener implements ServletContextListener {
-        public static final String WEBAPP_CONFIG_LOC = "/WEB-INF/config";
-
         public void contextInitialized(ServletContextEvent servletContextEvent) {
             System.out.println("contextInitialized...");
             ServletContext cntx = servletContextEvent.getServletContext();
-            ServerContext.init(cntx.getContextPath(), cntx.getServletContextName(), cntx.getRealPath(WEBAPP_CONFIG_LOC));
             VersionUtil.initVersion(cntx);  // can be called multiple times, only inits on the first call
```

Comment:
I wasn't clear.  You misunderstood.  I linked here so you can see that ServerContext.init was called before VersionUtils.initVersion.  That's why your call to VersionUtil.getVersionInfo inside of ServerContext.init is not getting valid version info because it hasn't been initialized.
The code here is good.  Don't change it.